hashicorp/consul
# Backport of feat: Change global-read-only policy to non editable into release/1.15.x

### Description
PR is manually created to backport https://github.com/hashicorp/consul/pull/18602

![image](https://github.com/hashicorp/consul/assets/10027860/08ee918d-503f-4343-9080-573cd2c0ce0e)



### Testing & Reproduction steps
1. set up CONSUL_ACLS_ENABLE to 1 in session storage
2. set CONSUL_POLICY_COUNT = 10
3. go to Policies
4. log in with any securId
5. check list of items to see global-read-only policy
6. go to the policy details and check that form is non-editable


### Links
[JIRA](https://hashicorp.atlassian.net/browse/CC-5792)
[Figma](https://www.figma.com/file/oMJ1VqcRzb2EP1mqfaohDz/View-Builtin-Token?type=design&node-id=916-8806&mode=design&t=ZRNWWdS4Bb0fhced-0)


### PR Checklist
